static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-article-promo.js

```diff
@@ -32,16 +32,26 @@ define([
         this.dataLinkNames = '';
         this.idealOutcome = 'Email sign-up is increased';
 
+        var $articleBody = $('.js-article__body');
+
         this.canRun = function () {
             //Tests for fronts, editions optional.
             var host = window.location.host,
                 escapedHost = host.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&'), // Escape anything that will mess up the regex
                 urlRegex = new RegExp('^https?:\/\/' + escapedHost + '\/(uk\/|us\/|au\/|international\/)?([a-z-])+$', 'gi'),
                 browser = detect.getUserAgent.browser,
-                version = detect.getUserAgent.version;
+                version = detect.getUserAgent.version,
+                allArticleEls = $('> *', $articleBody),
+                lastFiveElsParas = true;
+
+            for (var i = allArticleEls.length - 1; i > allArticleEls.length - 6; i--) {
```

Comment:
this is maybe a little tricky to read and could be simplified to a single expression, with the paragraph check as a standalone function e.g. `lastFiveElsParas = allArticleEls.slice(allArticleEls.length - 5).every(isParagraph)`